src/success.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Success Page</title>
+    <!-- Firebase -->
+    <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-auth.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-firestore.js"></script>
+
+    <!-- TODO: Add SDKs for Firebase products that you want to use
+     https://firebase.google.com/docs/web/setup#available-libraries -->
+    <script src="https://www.gstatic.com/firebasejs/7.11.0/firebase-analytics.js"></script>
+
+    <script>
+        // Your web app's Firebase configuration
+        var firebaseConfig = {
+            apiKey: "AIzaSyB5IScRHMh_E2rV7sZ0vXi8a13WQtv9Kas",
```

Comment:
@habeeba-naaz `Security threat`: Please do not expose any API tokens. Replace it with `XXXXXXXXXXX`. Add a comment with the URL to get the API key.

Reply:
Okay sure..thank you for your advice @vinitshahdeo. I have some doubts regarding this issue..do I need to write client side code also along with server api? If so where should I do client side code?

Reply:
@@habeeba-naaz Write a function to verify jwt token and to refresh the token if it expired or revoked.
Take a look at [this](https://firebase.google.com/docs/auth/admin/verify-id-tokens) to verify token.

Reply:
Okay I will check...I am understanding how to do from backend side?

Reply:
> Okay I will check...I am not understanding how to do from backend side?

You will get a jwt token from frontend so just create a function to verify the token and then parse it.
Use firebase admin sdk to  verify.
[Here's](https://firebase.google.com/docs/admin/setup) how to setup Firebase Admin SDK.

Reply:
Okay.... thank you...I’ll check